src/logistration/RegistrationPage.jsx

```diff
@@ -142,6 +157,16 @@ class RegistrationPage extends React.Component {
   }
 
   render() {
+    if (this.state.institutionLogin) {
+      return (
+        <InstitutionLogistration
+          onSubmitHandler={this.handleInstitutionLogin}
+          secondaryProviders={this.props.thirdPartyAuthContext.secondaryProviders}
```

Comment:
Will this not cause issue if component has not mounted yet and you try to access it? since `thirdPartyAuthContext` will be undefined by then?

Reply:
Nope this code would be executed when `this.state.institutionLogin` is true. it's value by default is false and only true when we click on the button `use my uni info` and that button only appears when thirdPartyContext is not empty.